sound/soc/bcm/digidac1-soundcard.c

```diff
@@ -359,10 +359,10 @@ static struct snd_soc_dai_link digidac1_soundcard_dai[] = {
 
 /* audio machine driver */
 static struct snd_soc_card digidac1_soundcard = {
-	.name         = "digidac1-soundcard",
-	.owner        = THIS_MODULE,
-	.dai_link     = digidac1_soundcard_dai,
-	.num_links    = ARRAY_SIZE(digidac1_soundcard_dai),
+	.nam		= "digidac1-soundcard",
```

Comment:
Careful with the reformatting.

It's a good idea to check your code still compiles and works before pushing.